assets/platforms/aruba_os.yaml

```diff
@@ -0,0 +1,57 @@
+---
+platform-type: 'aruba_os'
```

Comment:
is there a more specific name/kind for this? I know in scrapli community for python we have aruba_aoscx -- is that the same thing?

Reply:
I'm not sure if aruba_aoscx is `aruba_osswitch`? basically, `aruba_os` in PR is reprensent aruba wlan AC product line. if `aruba_aosxc` is equal to `aruba_os`, I can change the name to keep platform name consistency

Reply:
does [this](https://www.arubanetworks.com/techdocs/central/2.5.7/content/nms/policy/sup-device.htm) help at all? I have no idea about anything Aruba. its not a huge deal obviously but id like to try to have the naming consistent with the python bits and also just consistent in style. if this is for the wireless bits and there isn't a more specific name we could do `aruba_wlc` or something like that (this is what one of the fortinet platforms does).

Reply:
since I dont have any context for scrapli for python version. I don't know the platform rule here. I can give you some context for platform in `netmiko`:
`aruba_os`:  aruba AOS used for wireless controller
`aruba_osswitch`: basically for generic aruba network devices, like switch

Reply:
ok understood. lets roll w/ aruba_wlc just to differentiate it from the switches if that works for you?